src/app/thread/[threadId]/post/create/components/post-create-tab-list.tsx

```
@@ -0,0 +1,14 @@
+'use client';
+
+import { TabsList, TabsTrigger } from '@/components/ui/tabs';
+import { useLocale } from '@/app/contexts/index'
+
+export const PostCreateTabList = ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)